router.go

```diff
@@ -611,6 +611,29 @@ func respondIncidentHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+func round(val float64) int {
+	intVal := int(val)
+	if val-float64(intVal) > 0.5 {
+		return intVal + 1
+	}
+	return intVal
+}
+
+func getBearing(lat1 float64, lon1 float64, lat2 float64, lon2 float64) string {
```

Comment:
Realistically this is the only important part of this PR. We should probably take different parameters though.